app/assets/javascripts/components/samples.jsx

```diff
@@ -74,12 +85,38 @@ class Samples extends React.Component {
   applyClass(status) {
     if(status === 'COMPLETE') {
       return 'complete';
-    } else if (status === 'UPLOADING' || status === 'IN PROGRESS') {
-      return 'uploading';
+    } else if (status === 'WAITING') {
+      return 'waiting';
     } else if (status === 'INITIALIZING') {
       return 'initializing';
-    } else {
+    } else if (status === 'FAILED') {
+      return 'failed';
+    }
+  }
+
+  applyChunkStatusClass(postProcess, gsnapAlignment, hostFiltering) {
+    if (postProcess === 'LOADED' || gsnapAlignment ===   'LOADED' || hostFiltering === 'LOADED') {
```

Comment:
it should just be called Alignment because there're two types of alignments: gsnapl/rapsearch2.